client/src/main/scala/drt/client/components/FlightComponents.scala

```diff
@@ -5,17 +5,16 @@ import drt.shared._
 import drt.shared.api.Arrival
 import japgolly.scalajs.react.vdom.html_<^._
 import japgolly.scalajs.react.vdom.{TagOf, VdomArray}
-import org.scalajs.dom.html.Div
+import org.scalajs.dom.html.{Div, Span}
 
 
 object FlightComponents {
 
   def paxComp(flightWithSplits: ApiFlightWithSplits): TagMod = {
     val isNotApiData = if (flightWithSplits.hasValidApi) "right" else "right notApiData"
     <.div(
-      ^.title := paxComponentTitle(flightWithSplits.apiFlight),
       ^.className := s"$isNotApiData",
-      flightWithSplits.apiFlight.bestPaxEstimate
+      Tippy.describe(paxComponentDescription(flightWithSplits.apiFlight), flightWithSplits.apiFlight.bestPaxEstimate)
```

Comment:
`paxComponentDescription` suggests it's a description of a pax component. I reckon we should reverse the order of the latter two words to `paxDescriptionComponent`

Reply:
I'm going with paxNumberSources seeing as it's neither a description nor a component.